documentation/src/main/asciidoc/topics/proc_server_testing_datasources.adoc

```diff
@@ -0,0 +1,27 @@
+[id='testing_datasources']
+= Testing Data Sources
+Data Sources can be tested via the CLI to verify that the connection to the database is functioning correctly.
```

Comment:
```suggestion
Verify that connections to data sources are functioning correctly with the CLI.
```

Reply:
avoid passive voice and put focus on the user goal which is to "verify data source connection" rather than test via the CLI.